apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/ErrorReporting/ErrorReportingExceptionFilter.cs

```diff
@@ -70,15 +70,32 @@ public class ErrorReportingExceptionFilter : IExceptionFilter, IDisposable
         public static ErrorReportingExceptionFilter Create(string projectId, string serviceName, string version,
             ErrorReportingOptions options = null)
         {
-            GaxPreconditions.CheckNotNullOrEmpty(projectId, nameof(projectId));
```

Comment:
I split up the constructors a little to make this statement true as it really should be required in this case.